calibration/DOSTAM/CGINS-DOSTAM-00364__20201210.csv

```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+364,CC_calibration_date,1607558400000,sensor calibration date (milliseconds since 1970-01-01); source_file DOSTA-M_Optode-4831_SN_364_Calibration_2018-02-16.pdf

```

Comment:
milliseconds is correct but source_file is incorrect (I think we decided that we do not need to list them for these since no cal coefficients are actually being used?)